content/en/docs/languages/python/libraries.md

```diff
@@ -0,0 +1,109 @@
+---
+title: Using instrumentation libraries
+linkTitle: Libraries
+weight: 40
+cSpell:ignore: HTTPX httpx instrumentor uninstrument
+---
+
+{{% docs/languages/libraries-intro "Python" %}}
+
+## Use instrumentation libraries
+
+If a library isn't included with OpenTelemetry, you can use
```

Comment:
"If a library isn't included with OpenTelemetry" is a bit ambiguous to me, what about "If a library does not ship with native OpenTelemetry support"?

Reply:
Like it! Added.